tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2082,19 +2082,56 @@ public function theAdministratorHasAddedTheFollowingUsersToAGroupAtOnceUsingTheG
 	 * @throws GuzzleException
 	 * @throws Exception
 	 */
-	public function theAdministratorAddGroupToAGroupAtOnceUsingTheGraphApi(string $user, string $groupToAdd, string $group) {
+	public function theAdministratorTriesToAddGroupToAGroupAtOnceUsingTheGraphApi(string $user, string $groupToAdd, string $group) {
 		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");
 		$groupIdToAdd = $this->featureContext->getAttributeOfCreatedGroup($groupToAdd, "id");
 		$credentials = $this->getAdminOrUserCredentials($user);
 
+		$payload = [
+			"members@odata.bind" => [GraphHelper::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupIdToAdd)]
+		];
+
 		$this->featureContext->setResponse(
-			GraphHelper::addGroupToGroup(
-				$this->featureContext->getBaseUrl(),
+			HttpRequestHelper::sendRequest(
+				GraphHelper::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupId),
 				$this->featureContext->getStepLineRef(),
+				'PATCH',
 				$credentials["username"],
 				$credentials["password"],
-				$groupId,
-				$groupIdToAdd
+				['Content-Type' => 'application/json'],
+				\json_encode($payload)
+			)
+		);
+	}
+
+	/**
+	 * @When /^the administrator "([^"]*)" tries to add a group "([^"]*)" to another group "([^"]*)" with POST request using the Graph API$/
+	 *
+	 * @param string $user
+	 * @param string $groupToAdd
+	 * @param string $group
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function theAdministratorTriesToAddAGroupToAGroupThroughPostRequestUsingTheGraphApi(string $user, string $groupToAdd, string $group) {
+		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");
+		$groupIdToAdd = $this->featureContext->getAttributeOfCreatedGroup($groupToAdd, "id");
+		$credentials = $this->getAdminOrUserCredentials($user);
+
+		$payload = [
+			"@odata.id" => GraphHelper::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupIdToAdd)
+		];
+
+		$this->featureContext->setResponse(
+			HttpRequestHelper::post(
+				GraphHelper::getFullUrl($this->featureContext->getBaseUrl(), 'groups/' . $groupId . '/members/$ref'),
+				$this->featureContext->getStepLineRef(),
+				$credentials["username"],
+				$credentials["password"],
+				['Content-Type' => 'application/json'],
+				\json_encode($payload)
```

Comment:
Will it make sense to make this and the above step into one? The only difference I see is the URL which maybe we can switch if according to the request. 
```
@When /^the administrator "([^"]*)" tries to add a group "([^"]*)" to another group "([^"]*)" with "([^"]*)" request using the Graph API
```